Update community_lessons.json

annajiat opened the pull request:
Proposed:
http://hpc-carpentry.org/hpc-intro/
which will auto redirect to https://carpentries-incubator.github.io/hpc-intro/
because of hpc-carpentry.org/pages/hpc-intro.md

Current 404:
http://hpc-carpentry.github.io/hpc-intro/

psteinb commented:
I see you reviewed it, but I can still not merge. What does it say for your account?

ocaisa commented:
I'm not on the list of approving reviewers so that's a blocker

psteinb commented:
@symulation @razoumov Can you help @ocaisa, please?

ChristinaLK commented:
Based on @ocaisa's review, I can approve, but we should definitely get him the right permissions to approve himself!

annajiat committed:
Update community_lessons.json

Proposed:
http://hpc-carpentry.org/hpc-intro/
which will auto redirect to https://carpentries-incubator.github.io/hpc-intro/
because of hpc-carpentry.org/pages/hpc-intro.md

Current 404:
http://hpc-carpentry.github.io/hpc-intro/

ocaisa commented:
Thanks for this, it will also (I think) get magically fixed if we implement #8

ocaisa commented:
I'd leave this open for now until we do that, and then I would update all the links at once

Commit:
Update all links to use hpc-carpentry.org

ocaisa commented:
@annajiat I updated this to update all the other links as well since I renamed the repo, let me know that you are ok with it.

ocaisa commented:
Thanks @annajiat for checking this
